README.md

```diff
@@ -107,3 +107,12 @@ The following will mount /aqua-ml-data on your S drive and allow you to explore,
 ### setup environment variable
 The following will auto-select the bucket for local silnlp operations
 * Set the environment variable SIL_NLP_DATA_PATH to /aqua-ml-data
+
+## Setup ClearML on local PC
+Install the clearml python package:
+
+`pip install clearml`
```

Comment:
Doesn't `poetry install` install clearml?

Reply:
Yes, but this is setting up clearml globally (not just the virtual environment).  It was needed when I installed it.